.github/workflows/build-processor.yml

```
@@ -0,0 +1,39 @@
+name: Build and release processor
+
+on:
+  push:
+    tags:
+      - "processor-*"
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

[Node.js 16 actions are deprecated](https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/) (which `v3` uses), lets upgrade to `v4`
```suggestion
        uses: actions/checkout@v4
```